diff --git a/lib/control.sh b/lib/control.sh
old mode 100644
new mode 100755
index d8ceb9d898ddd4eec0c3ddca22e890b5b228e2df..b135de1fcbd4dbc0b7bb2b8a7f1f097d3ea7993a
--- a/lib/control.sh
+++ b/lib/control.sh
@@ -109,7 +109,7 @@ perform_build_steps()
     notice "enabled build steps (not in order): ${!build_steps[*]}"
 
     local step
-    for step in RETRIEVE CHECKOUT MANIFEST BUILD CHECK INSTALL_SYSROOT HELLO_WORLD TARSRC TARBIN; do
+    for step in CHECKOUT MANIFEST BUILD CHECK INSTALL_SYSROOT HELLO_WORLD TARSRC TARBIN; do
         if [ ! -z "${build_steps[$step]}" ]; then
 	    # this step is enabled
 	    notice "Performing build step $step"
